integrationTests/vm/txDataBuilder.go

```diff
@@ -0,0 +1,127 @@
+package vm
+
+import (
+	"encoding/hex"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+// TxDataBuilder constructs a string to be used for transaction arguments
+type TxDataBuilder struct {
+	Function  string
+	Elements  []string
+	Separator string
+}
+
+func NewTxDataBuilder() *TxDataBuilder {
+	return &TxDataBuilder{
+		Function:  "",
+		Elements:  make([]string, 0),
+		Separator: "@",
+	}
+}
+
+func (builder *TxDataBuilder) New() *TxDataBuilder {
+	builder.Function = ""
+	builder.Elements = make([]string, 0)
+	return builder
+}
+
+func (builder *TxDataBuilder) ToString() string {
+	data := builder.Function
+	for _, element := range builder.Elements {
+		data = data + builder.Separator + element
+	}
+
+	return data
+}
+
+func (builder *TxDataBuilder) ToBytes() []byte {
+	return []byte(builder.ToString())
+}
+
+func (builder *TxDataBuilder) Func(function string) *TxDataBuilder {
+	builder.Function = function
+	return builder
+}
+
+func (builder *TxDataBuilder) Byte(value byte) *TxDataBuilder {
+	element := hex.EncodeToString([]byte{value})
+	builder.Elements = append(builder.Elements, element)
+	return builder
```

Comment:
add empty space on all occurrences where we have something like:
```GO
instruction

return...
```

Reply:
Added.